### PR TITLE
Solved issue #145

### PR DESCRIPTION
**Pull request by Yash1256**
In these Screenshots the first is the list of all the commands which I used to install and use wplay, and the second screenshot contains the list of the tracking command

### CODE REVIEW

**Comment by xandao6** [comment]
Rename your screenshoots please, put your github username.

**Comment by Yash1256** [comment]
@xandao6 Renamed The files
Yash1256_ Activating Tracker
Yash1256_List of commands

**Comment by xandao6** [comment]
Yes, but you need to do a push to update the PR.

File: Contribution.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+1. Raise an issue before sending any PR.
+2. Make you changes to feature branch.
+3. See if there is already an open PR for the same issue.
+
+To create a launcher with arguments follow the steps bellow: <br />
+1. Click in 'Debug' tab
+1. Click in 'Add Configuration'
+1. Select 'Module'
+1. Type 'wplay' and press Enter
+1. A json file will be opened. Inside configurations add the args, for example: "args":["-wb","name"]
```
Comment (rpotter12) [inline Contribution.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922&bid=16766304)

File: Contribution.md (hunk 1 above)
Comment (rpotter12) [inline Contribution.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922)

**Comment by Yash1256** [comment]
Pushed the updated PR

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922&bid=16766304)

File: Contribution.md (hunk 1 above)
Comment (rpotter12) [inline Contribution.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922)

Comment (rpotter12) [inline Contribution.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922)

**Comment by rpotter12** [comment]
@Yash1256 You haven't updated your fork properly. 
Please update your fork forcefully and create a new PR :)

**Comment by Yash1256** [comment]
@rpotter12 Sir I have pulled and pushed again the required changes

**Comment by csharp489** [comment]
@Yash1256 , Can u please let me know how to do the PR. Actually I am also facing issue in this .Please let me know step by step process or any helpful link.

**Comment by priyansh19** [comment]
> @Yash1256 , Can u please let me know how to do the PR. Actually I am also facing issue in this .Please let me know step by step process or any helpful link.

@csharp489 have a look : https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request

**Comment by danilofuchs** [comment]
You need to do a rebase from your fork to this repository master and than do a push force onto your fork.
This might help: https://gist.github.com/ravibhure/a7e0918ff4937c9ea1c456698dcd58aa

Also, only include in your PR your changes, and not other people's changes, so avoid ever doing a merge commit into a PR's branch, as it confuses git. Always use rebase + push --force

File: Contribute.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+To create a launcher with arguments follow the steps bellow: <br />
+1. Click in 'Debug' tab
+1. Click in 'Add Configuration'
+1. Select 'Module'
+1. Type 'wplay' and press Enter
+1. A json file will be opened. Inside configurations add the args, for example: "args":["-wb","name"] 
+
+
+Make sure you follow below guidelines before contributing: <br />
```
Comment (rpotter12) [inline Contribute.md]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922&bid=16766304)

File: Contribute.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+## Contribute
+The first step is to create a fork and clone, then you can solve our [issues](https://github.com/rpotter12/whatsapp-play/issues) and help us find new ones. To debug with Visual Studio Code it is necessary to create a launcher with the arguments. <br />
+
+To create a launcher with arguments follow the steps bellow: <br />
+1. Click in 'Debug' tab
+1. Click in 'Add Configuration'
+1. Select 'Module'
+1. Type 'wplay' and press Enter
+1. A json file will be opened. Inside configurations add the args, for example: "args":["-wb","name"]
```
Comment (rpotter12) [inline Contribute.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922)

Comment (rpotter12) [inline Contribute.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922&bid=16766304)

File: Contribute.md (hunk 2 above)
Comment (rpotter12) [inline Contribute.md]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922)

File: Contribute.md (hunk 3 above)
Comment (rpotter12) [inline Contribute.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922)

Comment (rpotter12) [inline Contribute.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096922)

**Comment by rpotter12** [comment]
@Yash1256 you haven't updated your fork properly. Please update your fork forcefully by upstream. I suggest you do a google search on how to update ok our fork forcefully by upstream. :)

**Pull request closed without merging**